errai-security/errai-security-keycloak/src/main/java/org/jboss/errai/security/keycloak/KeycloakAuthenticationService.java

```diff
@@ -230,7 +230,17 @@ protected User createKeycloakUser(final AccessToken accessToken) {
   }
 
   private Collection<? extends Role> createRoles(final AccessToken accessToken) {
-    Set<String> roleNames = accessToken.getResourceAccess(accessToken.getIssuedFor()).getRoles();
+    Set<String roleNames = new HashSet<>();
```

Comment:
There are a few issues with this line:
* `Set<String` is missing a closing angle bracket.
* Errai 3.x targets Java 1.6 so we can't use the `<>` operator. It should be `<String>` instead.
* This is the first usage of `HashSet` in this file, so there's a missing import for `java.util.HashSet`.